docs/cep/tutorials/transforming-data.md

```diff
@@ -22,70 +22,61 @@ the temperature of a room every second are transformed to indicate the average t
 5. Add the following sample stream application.
 6. Let's define the input stream to define the schema based on which data is selected to the streaming integration flow.
 
-    In this example, let's assume that each event indicates the device ID, the room ID, and the temperature. Therefore, let's define an input stream as follows:
-
+    In this example, let's assume that each event indicates the device ID, the room number, and the temperature. Therefore, let's define an input stream as follows:
```

Comment:
Reading through this set of instructions, it seems like almost every use of "let's" is unnecessary. Try deleting them and see how it reads.